test/extensions/filters/http/rbac/rbac_filter_test.cc

```diff
@@ -46,11 +46,11 @@ class RoleBasedAccessControlFilterTest : public testing::Test {
     filter_.setDecoderFilterCallbacks(callbacks_);
   }
 
-  void setDestinationPort(uint16_t port, int times = 2) {
+  void setDestinationPort(uint16_t port, int at_most_times = 3) {
     address_ = Envoy::Network::Utility::parseInternetAddress("1.2.3.4", port, false);
     auto& expect = EXPECT_CALL(connection_, localAddress());
-    if (times > 0) {
-      expect.Times(times);
+    if (at_most_times > 0) {
```

Comment:
This isn't a useful expectation. Please just make this an ON_CALL(...).WillByDefault(...)

Reply:
Done